docs/pages/enterprise/introduction.mdx

```diff
@@ -120,8 +120,8 @@ cryptographic module (BoringCrypto) and fails to start if it was not.
 
 See our [FedRAMP for SSH and Kubernetes](ssh-kubernetes-fedramp.mdx) guide for more infromation.
 
-## Approval Workflows
+## Approval Requests
```

Comment:
let's rename Approval Workflows to Access Requests too, so there is only one term remaining